src/SectionTitle.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import cx from 'classnames';
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+export const SECTIONTITLE_CLASS = 'sectionTitle';
+/**
+ * @module SectionTitle
+ */
+class SectionTitle extends React.Component {
+	render() {
+		const {
+			action,
+			className,
+			title,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			SECTIONTITLE_CLASS,
+			className
+		);
+
+		return (
+			<Flex
+				align='center'
+				className={classNames}
+				{...other}
+				>
+				<FlexItem>
```

Comment:
Since chunk is in both would this structure still give the look we need

```
<Chunk>
  <FlexItem>
    ....
  </FlexItem>
  {action && <FlexItem />
</Chunk>